canonical-websites/www.canonical.com
# updating to marketo 2.0 form

## Done
- updated /services/contact-us to the marketo 2.0 form fields
- updated it to use the new (optional) format for required fields
## QA
1. go to /services/contact-us and see that the country/state field works
2. see that the form works
